wordpress-mobile/WordPress-Android
# Fix null siteModel when EditorMedia.start() is invoked

Fixes #10946 

Review + merge instructions
- wait until 13.8 gets merged into develop

Fixes two bugs
- crash when EditPostActivity is started without a SiteModel.
- EditorMedia.start is invoked before [this](https://github.com/wordpress-mobile/WordPress-Android/blob/590fc6bd5572363605e0c9c7e3f0911d328466af/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L430) and therefore is working with a wrong SiteModel.

~I haven't been able to reproduce the issue.~ The fix is pretty simply, however, I'm not sure it's safe. Both bugs were introduced in 13.8-rc-1. ~Do you think it's worth merging into 13.8 - @planarvoid @jkmassel @mzorz ?~ => I've decided it's not worth the risk. The QuickPress widget won't work in this release.

To test:
1. Add QuickPress widget to your desktop - link it to BlogA
2. Go to the app and switch to BlogB
3. Go to the desktop and click on the widget
4. Notice the app doesn't crash

PR submission checklist:
